### DISCUSSION TITLE
Should IP links be blocked?

### DISCUSSION THREAD

**Discussion by DecimalTurn** [DecimalTurn/comment-filter#5] [General]
While looking at other potential malware links, I found one that was using an IP address. Eg. http://91.XXX.XXX.XX/sb19rKQP
It seems rare and most people would know not to trust these, but it could still be worth blocking.

**Comment by cristianbuse** [comment]
First of all, thanks you! I fully intend to use this automation for my repos.

Because this is all about malicious intent, most users will perceive an IP address less dangerous and more transparent when compared to a shortened URL simply because the latter is obfuscated. But, in terms of inherent danger, neither one is more dangerous than the other, obviously.

Still, I would vote to block the IP addresses. I enjoy the mental comfort that DNS provides 😄 - like you said, people don't trust these IP addresses anyway.

**Reply by DecimalTurn** [reply]
Sounds good, thanks for the feedback. I've addred that to the latest release (as well as Discord CDN which is their new favorite method apparently).
